wagtail/admin/templates/wagtailadmin/shared/header.html

```diff
@@ -8,19 +8,24 @@
     - `search_url` - if present, display a search box. This is a URL route name (taking no parameters) to be used as the action for that search box
     - `query_parameters` - a query string (without the '?') to be placed after the search URL
     - `icon` - name of an icon to place against the title
+    - `avatar` - if present, display an 'avatar' in place of icon. This is the URL to be used as the img src for avatar 
```

Comment:
Sorry. Forgot to mention. We should add `classname` here as it will now be a supported variable